metadata-service/factories/src/main/java/com/linkedin/metadata/boot/factories/BootstrapManagerFactory.java

```diff
@@ -27,7 +27,7 @@ public class BootstrapManagerFactory {
   @Qualifier("entityService")
   private EntityService _entityService;
 
-  @Autowired
+  @Autowired(required = false)
```

Comment:
Yeah it should. Each step should have its own factory but nonetheless the step shouldn't be depending on Ebean specifically. This seems dangerous

Reply:
@Autowired(required = false) basically just makes it not force wiring, if it's there it's there, if it's not then it's usually accompanied by null checks/applicationContext checks. Since this class doesn't actually use the object, I don't think this should block the PR for another day as it should not cause any problems, but we should probably just remove it fully after this goes in.

Reply:
Missed it getting passed into the constructor here, yeah this will cause that step to fail if it's not present. We should pull this functionality out directly into the EntityService rather than have this step directly accessing the EbeanServer.

Reply:
Yeah, it would be best if it just depended on the EntityService. Should it be sorted in this PR?

Reply:
We have decided to address this on our side in a followup. No need to churn you kind folks on it.